tests/test_cli.py

```diff
@@ -43,6 +42,8 @@ class TestException(Exception):
     Custom exception we can throw for testing.
     """
 
+    __test__ = False
```

Comment:
What's happening here?

Reply:
Telling pytest that isn't a test class. Added a comment.